enlighten/parser/ExportFileParser.py

```diff
@@ -414,6 +426,10 @@ def process_data(self, values):
                     if len(value) == 0:
                         continue
 
+                    # MZ: honestly not sure how to handle these
+                    if value == "NA":
+                        value = 0
```

Comment:
[Suggestion] If it's a float you could use `float('NaN')`. Doesn't work for ints.